_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,465 @@
+            </select>
+        </div>
+        <div class="field">
+            <label class="field-label" for="select_02_err">Choose your favorite color of the rainbow (enhanced select menu with error)</label>
+            <select class="field-input input-select replace-select has-error" name="select_02_err" id="select_02_err">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+        <div class="field">
+            <label class="field-label" for="select_02_suc">Choose your favorite color of the rainbow (enhanced select menu with valid selection)</label>
+            <select class="field-input input-select replace-select has-success" name="select_02_suc" id="select_02_suc">
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+        <div class="field">
+            <label class="field-label" for="select_02_suc">Choose your favorite color of the rainbow (enhanced select and disabled menu)</label>
+            <select class="field-input input-select replace-select" name="select_02_suc" id="select_02_suc" disabled>
+                <option value="">- Choose one -</option>
+                <option value="Red">Red</option>
+                <option value="Orange">Orange</option>
+                <option value="Yellow">Yellow</option>
+                <option value="Green">Green</option>
+                <option value="Blue">Blue</option>
+                <option value="Indigo">Indigo</option>
+                <option value="Violet">Violet</option>
+            </select>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6 example-set-hd">Radios</h4>
+<div class="example-set">
+    <fieldset class="field-group">
+        <legend class="field-group-hd field-label">How would you prefer to be contacted? (default radio buttons)</legend>
+        <div class="field">
+            <label class="field-label label-inline" for="updates_mobile_text_01">
+                <input type="radio" class="field-input input-radio" name="updates_notifications_01" id="updates_mobile_text_01" value="Mobile text messages" aria-describedby="updates_mobile_text_01_helper" checked="checked">
```

Comment:
NIT: Let's keep this consistent with our attribute standards (and what you're doing with required fields/attrs) by just using the ``checked`` notation here. Cool?